legal-api/src/legal_api/resources/v2/business/business_tasks.py

```diff
@@ -70,10 +70,11 @@ def get_tasks(identifier):
         elif has_temp_reg_filing_todo:
             rv.append(create_temp_reg_filing_todo(temp_reg_filing, 1, True))
     else:
-        rv = construct_task_list(legal_entity)
+        rv = construct_task_list(business)
         if not rv and is_nr:
+            business_id = business.id if business.is_legal_entity else business.legal_entity_id
             paid_completed_filings = Filing.get_filings_by_status(
-                legal_entity.id, [Filing.Status.PAID.value, Filing.Status.COMPLETED.value]
+                business_id, [Filing.Status.PAID.value, Filing.Status.COMPLETED.value]
             )
```

Comment:
you'll need to update `Filing.get_filings_by_status` to work with a business(alternate name or legal entity) instead of business_id.

Reply:
updated